tests/express-mock.ts

```diff
@@ -11,27 +11,25 @@ vi.mock("express-fileupload", () => ({ default: fileUploadMock }));
 const staticHandler = vi.fn();
 const staticMock = vi.fn(() => staticHandler);
 
-let appMock: Record<
-  "disable" | "use" | "get" | "post" | "put" | "patch" | "delete" | "options",
-  Mock
->;
-
-const expressMock = () => {
-  appMock = {
-    disable: vi.fn(() => appMock),
-    use: vi.fn(() => appMock),
-    get: vi.fn(),
-    post: vi.fn(),
-    put: vi.fn(),
-    patch: vi.fn(),
-    delete: vi.fn(),
-    options: vi.fn(),
-  };
-  return appMock;
+const appMock = {
+  disable: vi.fn(() => appMock),
+  use: vi.fn(() => appMock),
+  get: vi.fn(),
+  post: vi.fn(),
+  put: vi.fn(),
+  patch: vi.fn(),
+  delete: vi.fn(),
+  options: vi.fn(),
+  init: vi.fn(),
 };
```

Comment:
this is a better implementation, should be cherry-picked to master along with `afterEach` in server test.

Reply:
addressed in #2103